pkg/pubsub/adapter/converters/scheduler.go

```diff
@@ -73,7 +74,8 @@ func convertScheduler(ctx context.Context, msg *cepubsub.Message, sendMode ModeT
 		for k, v := range msg.Attributes {
 			// CloudEvents v1.0 attributes MUST consist of lower-case letters ('a' to 'z') or digits ('0' to '9') as per
```

Comment:
I would also leave the previous comment... same in other places

Reply:
Hmm, I did not delete the previous comment.. Or do mean something else?

Reply:
Ohh you are right, is not in red... my bad..